test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -119,4 +119,20 @@ declare module 'test-case-reporting' {
       time: number; // in milliseconds
     }
   }
+
+  namespace Travis {
+    export interface Report {
+      job: Travis.Job;
+      build: Travis.Build;
+      result: KarmaReporter.TestResultReport;
+    }
+    export interface Build {
+      buildNumber: string;
+      commitSha: string;
+    }
+    export interface Job {
+      jobNumber: string;
```

Comment:
Should this be type number?

Reply:
Job numbers look like `1234.567`, so it should be a string.

Reply:
SG

Reply:
Although that should be fine with number type too...

Reply:
We might get a floating point error in that case, I believe.

Reply:
I'd stick with string here since JS floating points can drift, and since it's not actually representative of a number